_config.yml

```diff
@@ -3,3 +3,4 @@ title: Cloud Spanner Ecosystem
 description: An open source community for Cloud Spanner
 github:
   is_project_page: false
+  is_user_page: false
```

Comment:
I don't think this is used in the Slate template (https://github.com/pages-themes/slate/blob/master/_layouts/default.html). (Not sure why `is_project_page` isn't respected.)

Reply:
Yeah, you're right.

It looks like jekyll is picking up an old version of slate from before the PR that uses `is_project_page`.

Closing this PR.

Reply:
https://github.com/pages-themes/slate/issues/24#issuecomment-577523894